### ISSUE TITLE
rule terraform_standard_module_structure throws false warnings

### ISSUE THREAD

**Issue by nope-no** [terraform-linters/tflint#834]
```hcl
# .tflint.hcl
rule "terraform_standard_module_structure" {
  enabled = true
}
```
Maybe I'm holding it wrong but shouldn't the following examples give no warnings?
```bash
$ tflint ok
2 issue(s) found:

Warning: output "test" should be moved from ok/outputs.tf to outputs.tf (terraform_standard_module_structure)

  on ok/outputs.tf line 1:
   1: output "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: variable "test" should be moved from ok/variables.tf to variables.tf (terraform_standard_module_structure)

  on ok/variables.tf line 1:
   1: variable "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

$ tflint ok/
3 issue(s) found:

Warning: Module should include a main.tf file as the primary entrypoint (terraform_standard_module_structure)

  on ok/main.tf line 1:

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: output "test" should be moved from ok/outputs.tf to outputs.tf (terraform_standard_module_structure)

  on ok/outputs.tf line 1:
   1: output "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: variable "test" should be moved from ok/variables.tf to variables.tf (terraform_standard_module_structure)

  on ok/variables.tf line 1:
   1: variable "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

$ tflint dir_that_only_contains_subdirs
3 issue(s) found:

Warning: Module should include a main.tf file as the primary entrypoint (terraform_standard_module_structure)

  on dir_that_only_contains_subdirs/main.tf line 1:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: Module should include an empty outputs.tf file (terraform_standard_module_structure)

  on dir_that_only_contains_subdirs/outputs.tf line 1:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: Module should include an empty variables.tf file (terraform_standard_module_structure)

  on dir_that_only_contains_subdirs/variables.tf line 1:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

```
(If I cd into ok and run tflint everything is fine.)

## Version
TFLint v0.17.0

Many thanks in advance. Keep up the excellent work.

**Comment by bendrucker** [comment]
Can I get some example configuration here so I can see the project structure?

**Comment by nope-no** [comment]
Sure.

The directory ok contains 3 files:

```hcl
# main.tf
provider "aws" {
  version = "~> 2.0"
  region  = "us-east-1"
}
resource "aws_vpc" "test" {
  cidr_block = "10.0.0.0/16"
}
```

```hcl
# variables.tf
variable "test" {
  default = "test"
}
```

```hcl
# outputs.tf
output "test" {
  value = "test"
}
```

**Comment by bendrucker** [comment]
And how are you invoking TFLint? In other words, the command?

**Comment by nope-no** [comment]
```bash
$ /usr/local/bin/tflint -c /tmp/.tflint.hcl /tmp/ok
2 issue(s) found:

Warning: output "test" should be moved from /tmp/ok/outputs.tf to outputs.tf (terraform_standard_module_structure)

  on /tmp/ok/outputs.tf line 1:
   2: output "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

Warning: variable "test" should be moved from /tmp/ok/variables.tf to variables.tf (terraform_standard_module_structure)

  on /tmp/ok/variables.tf line 1:
   1: variable "test" {

Reference: https://github.com/terraform-linters/tflint/blob/v0.17.0/docs/rules/terraform_standard_module_structure.md

```

**Comment by bendrucker** [comment]
Got it, that seems like a bug with that rule, it should be resolving the filename relative to the module dir. Will look into it soon, happy to review a PR too.
